### PR TITLE
Fix a few spelling items and adjust language

### PR DESCRIPTION
**Pull request by szaydel**
This is a PR with just the few minor mistakes I noticed and a couple of changes to language which I felt might be helpful.

### CODE REVIEW

File: series/series.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func (e boolElements) Len() int           { return len(e) }
 func (e boolElements) Elem(i int) Element { return &e[i] }
 
 // ElementValue represents the value that can be used for marshaling or
-// unmarshaling Elements.
+// un-marshaling Elements.
```
Comment (kniren) [inline series/series.go]: The previous spelling was correct: https://en.wikipedia.org/wiki/Unmarshalling

File: dataframe/dataframe.go [hunk 2]
```diff
@@ -761,7 +761,7 @@ func WithDelimiter(b rune) LoadOption {
 
 // LoadStructs creates a new DataFrame from arbitrary struct slices.
 //
-// LoadStructs will ignore unexported fields inside an struct. Note also that
+// LoadStructs will ignore un-exported fields inside an struct. Note also that
```
Comment (kniren) [inline dataframe/dataframe.go]: The previous spelling was correct

File: dataframe/dataframe.go [hunk 3]
```diff
@@ -582,7 +582,7 @@ func (df DataFrame) Capply(f func(series.Series) series.Series) DataFrame {
 }
 
 // Rapply applies the given function to the rows of a DataFrame. Prior to applying
-// the function the elements of each row are casted to a Series of a specific
+// the function the elements of each row are cast to a Series of a specific
```
Comment (kniren) [inline dataframe/dataframe.go]: I'm torn on this one, I believe that `casted` is correct in this situation. Trying to find some consensus I arrived here: https://english.stackexchange.com/questions/94565/can-casted-be-the-past-tense-of-cast

Comment (szaydel) [reply dataframe/dataframe.go]: I think purely grammatically I hate to see casted. It is equivalent to someone saying: "Casted a spell" as opposed to "Cast a spell", but I am sure I can live with either.

File: series/series.go (hunk 1 above)
Comment (szaydel) [reply series/series.go]: And I can address this and will target development branch. I am guessing the intent for master is to be fairly stable.

Thanks a lot!

**Pull request closed without merging**